### PR TITLE
updated docs with recently added extended memory drivers

### PR DESCRIPTION
**Pull request by mrdudz**
what the topic says :)

### CODE REVIEW

File: doc/c128.sgml [hunk 1]
```diff
@@ -191,11 +191,19 @@ missing on VDC, and are translated to the two colors missing from the VIC palett
 
 <descrip>
 
+  <tag><tt/c128-efnram.emd (c128_georam_emd)/</tag>
```
Comment (oliverschmidt) [inline doc/c128.sgml]: C&P error at the static driver name

File: doc/c128.sgml [hunk 2]
```diff
@@ -191,11 +191,19 @@ missing on VDC, and are translated to the two colors missing from the VIC palett
...
+  Extended memory driver for the C128 External Function RAM.
+  Written and contributed by Marco van den Heuvel.
+
   <tag><tt/c128-georam.emd (c128_georam_emd)/</tag>
   A driver for the GeoRam cartridge. The driver will always assume 2048 pages
   of 256 bytes each. There are no checks, so if your program knows better,
   just go ahead.
 
+  <tag><tt/c128-ifnram.emd (c128_georam_emd)/</tag>
```
Comment (oliverschmidt) [inline doc/c128.sgml]: C&P error at the static driver name

**Comment by mrdudz** [comment]
fixed :)

**Comment by oliverschmidt** [comment]
Thanks :-)

**Pull request merged**